aries-site/src/examples/templates/dashboards/content/UpdatesAvaliable.js

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from 'react';
+import { NotificationItem, UpdatesFeed } from '../components';
+
+const MOCK_DATA = require('../../../../data/mockData/firmware-bundles.json');
+
+export const UpdatesAvaliable = () => {
+  const data = MOCK_DATA['firmware-bundles'].items;
+  const [update, setUpdate] = useState([]);
+
+  useEffect(() => {
+    if (data) {
+      const updateArray = [];
+      const sortedDates = data.sort((a, b) => {
+        const c = new Date(a.releaseDate);
+        const d = new Date(b.releaseDate);
+        return d - c;
+      });
+      const base = sortedDates.filter(updates => updates.type === 'base');
+      const hotfix = sortedDates.filter(updates => updates.type === 'hotfix');
+      updateArray.push(hotfix[0], base[0]);
+      setUpdate(updateArray);
+    }
+  }, [data]);
+
+  return (
+    <UpdatesFeed background="validation-warning" title="Updates Available!">
```

Comment:
Not for this PR. But I think we can make this more generic and turn into a "DashboardCard" which would provide the consistent wrapper throughout. We have some naming conflicts and other clean up, but this is an opportunity for us to simplify and reuse.

Reply:
Agree I will go through and change some names but I think I'll go back and clean up how we are using the different files and spend more time on how we can reconfigure them to be reusable. This was next on my list